src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -46,28 +50,44 @@ public Line findLineById(Long id) {
         return lineRepository.findById(id).orElseThrow(RuntimeException::new);
     }
 
-
+    @Cacheable(value = "lines", key = "#id")
     public LineResponse findLineResponseById(Long id) {
         Line persistLine = findLineById(id);
         return LineResponse.of(persistLine);
     }
 
+    @Caching(evict = {
+            @CacheEvict(value = "lines", key = "#id"),
+            @CacheEvict(value = "lines", allEntries = true)
```

Comment:
```suggestion
            @CacheEvict(value = "lines", allEntries = true)
```

`allEntries`를 조건으로 evict을 해줬다면 key 조건은 제거해도 되지 않을까요?

Reply:
아 맞네요. 중간에 키를 변경하면서 누락한 내용인 것 같습니다 👍